packages/elements/src/list/index.ts

```diff
@@ -649,17 +644,14 @@ export class List<T extends DataItem = ItemData> extends ControlElement {
     this.addEventListener('focusout', this.onBlur);
   }
 
-  protected update (changeProperties: PropertyValues): void {
+  /**
+   * Invoked before update() to compute values needed during the update.
+   * @param changeProperties changed properties
+   * @returns {void}
+   */
+  protected willUpdate (changeProperties: PropertyValues): void {
```

Comment:
Call super?

Reply:
No, It is not required in both cases.

Reply:
Yes, apparently so, https://lit.dev/docs/components/lifecycle/#reactive-update-cycle-performing. We should try using `willUpdate()` instead of `update()` where possible.